superset/connectors/sqla/models.py

```diff
@@ -649,7 +649,7 @@ def get_sqla_query(  # sqla
             elif m in metrics_dict:
                 metrics_exprs.append(metrics_dict.get(m).get_sqla_col())
             else:
-                raise Exception(_("Metric '{}' is not valid".format(m)))
+                raise Exception(_(f"Metric '{m}' does not exist"))
```

Comment:
It was wrong before, but variable formatting with `get_text` should use a different formatting scheme. Should look something like `_("Metric '%(metric)s' does not exist", metric=metric)`
https://pythonhosted.org/Flask-Babel/#flask.ext.babel.gettext